core/lib/server/form.ts

```diff
@@ -48,6 +49,11 @@ export const getForm = (
 					eb("forms.pubTypeId", "=", props.pubTypeId!).and("forms.isDefault", "=", true)
 				)
 			)
+			.$if(Boolean(props.pubId), (eb) =>
+				eb
+					.innerJoin("pubs", "pubs.pubTypeId", "forms.pubTypeId")
+					.where("pubs.id", "=", props.pubId!)
+			)
```

Comment:
this was to make the pub detail page load faster, now we can get the form and pub at the same time, rather than getting the pub first, then getting the form by its pubtype